RESIZE_IMAGE:
  WIDTH: 1200
EDGE_DETECTION:
  APERTURE: 3
  HIGH_THRESHOLD: 400
  LOW_THRESHOLD: 90
LINE_DETECTION:
  THRESHOLD: 150
  DIAGONAL_LINE_ELIMINATION: true
  DIAGONAL_LINE_ELIMINATION_THRESHOLD_DEGREES: 30
BORDER_REFINEMENT:
  LINE_WIDTH: 4
  WARPED_SQUARE_SIZE: [50, 50]
  NUM_SURROUNDING_SQUARES_IN_WARPED_IMG: 5
  SOBEL_KERNEL_SIZE: 3
  EDGE_DETECTION:
    HORIZONTAL:
      APERTURE: 3
      HIGH_THRESHOLD: 300
      LOW_THRESHOLD: 120
    VERTICAL:
      APERTURE: 3
      HIGH_THRESHOLD: 200
      LOW_THRESHOLD: 100
MAX_OUTLIER_INTERSECTION_POINT_RATIO_PER_LINE: 0.7
RANSAC:
  BEST_SOLUTION_TOLERANCE: 0.15
  OFFSET_TOLERANCE: 0.1
